bundles/automation/org.eclipse.smarthome.automation.core/src/main/java/org/eclipse/smarthome/automation/core/internal/RuleRegistryImpl.java

```diff
@@ -234,10 +234,10 @@ public Rule add(Rule rule) {
         String rUID = rule.getUID();
         if (rUID == null) {
             rUID = ruleEngine.getUniqueId();
-            super.add(initRuleId(rUID, rule));
-        } else {
-            super.add(rule);
+            rule = initRuleId(rUID, rule);
         }
+        ruleEngine.normalizeRuleConfigurations(rule);
```

Comment:
>Don't we already expect it to be normalised when being added to the registry

I think not, otherwise we don't need the normalization at all. 

Actually the problem is related with the default value of configuration property. It's value  is defined by its ModuleType as string, but this string should be normalize to the type defined by the ConfigDescriptionParameter of the property. The property can be  can be Decimal or  multiple and etc.